UserList/UserList/Controller/HomeViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  UserList
+//
+//  Created by Hoang Linh Nguyen on 25/8/2023.
+//
+
+import UIKit
+
+final class HomeViewController: UIViewController {
+    @IBOutlet private weak var searchBar: UISearchBar!
+    @IBOutlet private weak var userTableView: UITableView!
+    
+    private var reusableTableView: ReusebleTableView!
+    private var users = [User]()
+    private var items = [Item]()
+    private var tapIndex: Int?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        customizeView()
+        fetchAPI()
+    }
+    
+    private func customizeView(){
```

Comment:
```suggestion
    private func customizeView() {
```
Tương tự với các phần còn lại